### ISSUE TITLE
[ES - Bug] Las disciplinas no se eliminan a la primera

### ISSUE THREAD

**Issue by Pronormalnoob** [AnimaBeyondDevelop/AnimaBeyondFoundry#208]
### Información

- **Versión de Foundry: 11**
- **Versión del sistema: 1.18.1**
- **Navegador: Desktop**

### Descripción
No funciona bien el botón de eliminar cuando quiero quitar una disciplina psíquica de una ficha.

#### ¿Qué ocurre?
Al intentar eliminar una disciplina psíquica a veces no pasa nada, tenia Disciplinas Desconocidas por el cambio de versión, y al seguir intentándolo me parece que o funciona a la primera, o funciona a la tercera.

#### Contexto adicional
Me falla tanto con Disciplina Desconocida como con disciplinas oficiales.

**Comment by Miliviu** [comment]
Soy incapaz de replicar este problema, ¿Te sigue pasando? Si es así, ¿puedes mandar un video haciendolo/explicar con más detalle como replicarlo?

¡Gracias! @Pronormalnoob

**Comment by ORV-SalaScala** [comment]
No, solo me paso tras actualizar, así que no puedo volver a realizarlo (aunque la semana que viene seguramente actualice de nuevo).

**Comment by paiva92** [comment]
Entiendo que esto en la beta ya no sucede, yo no lo he podido replicar.
Lo comento ya que es un issue que se pasó a Testing.
Si alguien más puedo corroborar de que puede eliminar disciplinas psíquicas correctamente doy por cerrado el issue.

**Comment by Ryuoniak** [comment]
Testeado en la versión beta 2.0.0.10.

Yo también he podido eliminar sin ningún problema las disciplinas psíquicas de varias fichas.
